chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# t7 - get only the filenames
+set -eo pipefail
+
+
+# Function for printing error messages and exiting with a specific exit code
+# $1 - The exit code to use when exiting the script
+# $2 - The error message to print
+err() {
+  local code="$1"
+  shift
+  echo "[[ERROR]: $(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+  exit "$code"
+}
+
+
+# args check
+if [ "$#" -ne 1 ]; then
+  err 255 'Specify a single directory.'
+fi
+
+
+# Find only regular files and put in array.
+files=$(find "$1" -type f)
```

Comment:
Is `files` var used? I don't see that. I see this logic repetition in 29th line

Reply:
> Is `files` var used? I don't see that. I see this logic repetition in 29th line

My mistake, as in the 6th task, I redid it but did not remove the excess. Fixed